OloEngine/src/OloEngine/Renderer/Font.cpp

```
@@ -1,141 +1,163 @@
+            glyph.PlaneBoundsRight = static_cast<f32>(x1) * emScale;
+            glyph.PlaneBoundsTop = static_cast<f32>(y1) * emScale;
+
+            m_Data->Glyphs[codepoint] = glyph;
+            ++glyphCount;
         }
 
-        m_AtlasTexture = CreateAndCacheAtlas<u8, f32, 3, msdf_atlas::msdfGenerator>("Test", static_cast<f32>(emSize), m_Data->Glyphs, m_Data->FontGeometry, width, height);
+        // Load kerning pairs — use codepoint-based lookup to match GetAdvance().
+        // stbtt_GetKerningTable returns glyph indices, so we build a reverse
+        // mapping (glyphIndex → codepoint) for correct key construction.
+        if (const int kernTableLength = ::stbtt_GetKerningTableLength(&fontInfo); kernTableLength > 0)
```

Clang (14.0.6)

[warning] variable 'kernTableLength' is not initialized

(cppcoreguidelines-init-variables)